### DISCUSSION TITLE
Version 6 timestamp field auto update

### DISCUSSION THREAD

**Discussion by arsekil** [keystonejs/keystone#8413] [Questions]
Hello team,

I am trying to find a way to programmatically update a timestamp field upon every login. To keep a log of logins with updating timestamps.

Is it possible?
Did anyone try to to do similar?

Thanks

**Comment by marekryb** [comment]
I think best way to do this is to hook up into session management. So something like this should work:
```typescript
const session = statelessSessions({
  secret: yoursecret,
});

export default withAuth(
  config({
    ...
    session: {
      ...session,
      async start(args) {
        console.log("login", args.data);
        // do some stuff here
        return session.start(args);
      },
    },
    ...
  })
)
```
EDIT: statelessSessions.start can fail, so I think you have to check the result first and then do your custom stuff. But anyway you get the idea.
